### ISSUE TITLE
After hook in cucumber 

### ISSUE THREAD

**Issue by guillegette** [webdriverio/webdriverio#637]
Hi there,

Im trying to use the After hook from cucumber-js but it seems to break the wdio test runner.

```javascript
var AfterHooks = function () {
    this.After(function (callback) {
        console.log('after hook');
        callback();
    });
};
module.exports = AfterHooks;
```

And this is the output right after the console shows "after hook"
```
TypeError: Cannot call method 'replace' of undefined
  at [object Object].CucumberReporter.getUriOf (/Users/gette/Sites/webdriverio-test/node_modules/webdriverio/lib/frameworks/cucumber.js:142:26)
  at [object Object].CucumberReporter.handleStepResultEvent (/Users/gette/Sites/webdriverio-test/node_modules/webdriverio/lib/frameworks/cucumber.js:79:20)
  at process._tickCallback (node.js:419:13)
```

**Comment by christian-bromann** [comment]
Hi @guillegette ,
how do you apply that afterHook to WebdriverIO? Are you using the test runner?

**Comment by guillegette** [comment]
Hi @christian-bromann 

I placed that scrip inside features/support/hooks.js cucumber-js pick this file automatically. Yes Im using webio test runner and I can see the feature being executed but once it finishes the last step I can see the "after hook" in the console log and right after that TypeError

**Comment by christian-bromann** [comment]
I see, yeah looks like a bug in WebdriverIO. I will fix this asap and publish it later today. In the mean time maybe you can use one of the wdio internal hook: https://github.com/webdriverio/webdriverio/blob/master/examples/wdio.conf.js#L164-L192

Cheers

**Comment by guillegette** [comment]
Thanks @christian-bromann 

The hooks from wdio are great but they are after/before everything. The ones from cucumber-js are after/before a scenario which is a good place to do some housekeeping.

**Comment by christian-bromann** [comment]
fixed..will get shipped with v3.2.0

**Comment by christian-bromann** [comment]
Fixed and published with `v3.2.0`
